docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/05_CMS_Architecture.md

```diff
@@ -49,6 +52,17 @@ As there's a whole lot of CSS driving the CMS, we have certain best practives ar
 
 See our [system requirements](/getting_started/server_requirements) for a list of supported browsers.
 
+## Sprites
+
+We use sprites to handle various icons and images throughout the CMS. These are automatically generated
+by running `npm run compile` and can be found at `/admin/images/sprites/dist`. To add new
```

Comment:
Don't forget to update the naming here as well - I suggested `npm run css` instead of `npm run compile`. Might make sense to have a separate `npm run sprites` task.